examples/official-storybook/image-snapshots/storyshots-image.runner.js

```diff
@@ -21,13 +21,22 @@ if (!fs.existsSync(pathToStorybookStatic)) {
     suite: 'Image snapshots',
     framework: 'react',
     configPath: path.join(__dirname, '..'),
-    storyNameRegex: /^((?!tweaks static values with debounce delay|Inlines component inside story).)$/,
```

Comment:
What would be the correct way to ignore those stories?

Reply:
That's the way to ignore... But adding a proper ignore list to the api will definitely simplify everything...

Reply:
Looks like it didn't work: it ignored all the tests instead

UPD: I just missed `*` in the end, it had to be this:
```
/^((?!tweaks static values with debounce delay|Inlines component inside story).)*$/
```

Reply:
> adding a proper ignore list

#2679 should help a lot here. We could just introduce `skipSnapshots` story parameter (name can be configurable)

Reply:
Yeah 👍